Content.Server/_RMC14/Xenonids/Construction/EggMorpher/EggMorpherSystem.cs

```diff
@@ -0,0 +1,269 @@
+using Content.Shared._RMC14.Hands;
+using Content.Shared._RMC14.Xenonids.Construction.EggMorpher;
+using Content.Shared._RMC14.Xenonids.Egg;
+using Content.Shared._RMC14.Xenonids.Hive;
+using Content.Shared._RMC14.Xenonids.Parasite;
+using Content.Shared._RMC14.Xenonids.Projectile.Parasite;
+using Content.Shared.Coordinates;
+using Content.Shared.Database;
+using Content.Shared.Ghost;
+using Content.Shared.Hands.EntitySystems;
+using Content.Shared.Interaction;
+using Content.Shared.Mobs.Systems;
+using Content.Shared.Popups;
+using Content.Shared.StepTrigger.Systems;
+using Content.Shared.Stunnable;
+using Content.Shared.Verbs;
+using Robust.Shared.Network;
+using Robust.Shared.Player;
+using Robust.Shared.Timing;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Content.Server._RMC14.Xenonids.Construction.EggMorpher;
```

Comment:
All of this code can and should be in shared, so you don't need a separate system
If you need to change behavior on the client you can do net is client or is server checks